omegatrade-app/backend/app/helpers/stockdata.helper.js

```diff
@@ -0,0 +1,37 @@
+const { Spanner } = require('@google-cloud/spanner')
+
+/**
+* Returns a Spanner numeric Object.
+*/
+const spannerNumericRandVal = (value) => {
+    return Spanner.numeric(value.toString());
+}
+
+/**
+ * Returns a random number between min (inclusive) and max (inclusive)
+ */
+const spannerNumericRandValBetween = (min, max, scale = null) => {
+    const rand = Math.random() * (max - min) + min;
+    if (scale) {
+        const power = Math.pow(10, scale);
+        return Spanner.numeric((Math.floor(rand * power) / power).toString());
+    }
+    return Spanner.numeric(Math.floor(rand).toString());
```

Comment:
yes, this function returns spanner numeric contains decimal number only if the scale parameter is passed.
and it will return only spanner numeric contains integer if the scale is not passed.
for example:
while creating a stock data , `stockData.close` should lie between `dayHigh` and `dayLow`
and `stockData.volume` contains high-value integer data, so I've used numeric datatype for both `close` and `volume`
and inorder to reuse functions I've used the same function `spannerNumericRandValBetween`